client/templates/home.html

```diff
@@ -1,12 +1,32 @@
 <template name="home">
 
-  {{#if currentUser}}
-    {{> meetingSeriesList}}
+  {{#if sendVerificationEmailEnabled}}
+      {{#unless currentUser.emails.[0].verified}}
+          <p class="alert alert-warning">You need to verify your email address before.
```

Comment:
Maybe we should also think about the security aspect again. Should a not-verified user be able to view or manipulate a meeting series if he was added as invited or moderator. Currently this is possible if he knows the route to a meeting series. Should we simply forbid accessing the routes or even calling meteor methods?